addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1694,6 +1694,21 @@ body.editor_enable.editor_has_snippets {
                 background-color: $o-we-sidebar-bg;
             }
         }
+
+        div.o_we_invisible_element_is_root {
```

Comment:
I'm quite sure you can get rid of this class :eyes:

Reply:
Unfortunalty, I think that we can not get rid of this class. Indeed, we are targeting a `<div>` element that is followed by an `<ul>` element. If I am not mistaken there is no way to select an element preceding another in css.